workshop.pl

```diff
@@ -366,6 +367,7 @@ sub arg_handler {
 logger('debug', Dumper($target_json));
 
 my $tmp_container = $me . "_" . $target_json->{'label'};
+$tmp_container .= "_" . $args{'label'} if defined $args{'label'};
```

Comment:
I'm not sure I like this syntax.  I know it works just fine but it seems so backwards to me.  At the very least I think we should be consistent and I don't do this anywhere.

```
if (defined($args{'label'})) {
$tmp_container .= "_" . $args{'label'};
}
```

Reply:
I can change it.  Lately I have been trying a coding style which reduces the number of lines.

Reply:
Yeah, its clean and small but I think think it is horrible to read.

Reply:
Yeah, it's short and clean but I just find it so hard to read.

Reply:
Well that was dumb of me, I thought the first one got lost.